tests/integration/tls/test_tls.py

```diff
@@ -81,12 +137,15 @@ async def test_mongos_tls_disabled(ops_test: OpsTest) -> None:
         timeout=300,
     )
 
+    await check_mongos_tls_disabled(ops_test)
```

Comment:
necessary to fix flakey TLS tests - shouldn't check if disabled until model is stable